core/src/UserSettingsDialog.html

```diff
@@ -33,19 +36,38 @@
             saveBtn = usLocalization.saveBtn ? usLocalization.saveBtn : saveBtn;
             dismissBtn = usLocalization.dismissBtn ? usLocalization.dismissBtn : dismissBtn;
         }
+
         let key;
-        for (let k in userSettingsData[0]) {
+        for (let k in userSettingGroups[0]) {
             key = k;
             break;
         }
```

Comment:
```suggestion
       let key = Object.keys(userSettingGroups[0]).length > 0 ? Object.keys(userSettingGroups[0])[0] : undefined;
```

if we want to reduce number o lines... not really important

Reply:
always good to reduce number of lines 👍